packages/app/src/app/report/reportTerm.tsx

```
@@ -0,0 +1,73 @@
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+`;
+
+const MessageExampleImage = styled.Image`
+  width: 350px;
+  height: 362px;
+`;
+
+const ButtonContainer = styled.View`
```

ESLint

[error] 'ButtonContainer' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)